src/lib.rs

```diff
@@ -155,6 +155,12 @@ impl FourCharCode {
     pub fn display(&self) -> Display {
         Display(u32::from_be(normalize(self.0)))
     }
+
+    /// Returns the underlying `u32` this [FourCharCode] represents
+    #[inline]
+    pub fn to_u32(&self) -> u32 {
```

Comment:
The function need to be `const`, the `normalize` func is meant to be `const`. But, please, delete the normalization, we need to trust the struct state at this point for optimization purpose. In the end, according to [this](https://rust-lang.github.io/api-guidelines/naming.html#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv) documentation, the method need to be called `as_u32`

Reply:
Ah yep, const and naming are very sensible - just getting on a train, but I'll look into these when I get a chance!

Reply:
Yeah, I did it bad too when I was less experienced. Can you just rewrite the commit to keep the PR clean?

Reply:
Yep, will do 👍

Reply:
Cool, force-pushed that up, after double-checking I was happy that `from_bytes` was always producing a normalised value!

(tbf, it might be worth documenting that somewhere, though I'm not sure where is best to note it)